### ISSUE TITLE
Implement fastdom.sandbox()

### ISSUE THREAD

**Issue by wilsonpage** [wilsonpage/fastdom#56]
### Use Case

I'm writing a web-component and need to schedule a few dom related jobs. The web-component is destroyed and those jobs are still pending. I'd like to `.clear()` just the jobs related to that component.

```js
this.fastdom = fastdom.sandbox();

this.fastdom.read(...);
this.fastdom.write(...);
...

this.fastdom.clear();
```

**Comment by wilsonpage** [comment]
This is not the same as creating a brand new `FastDom` instance as the jobs scheduled via the sandbox should be run the in usual global batch.

**Comment by wilsonpage** [comment]
Implemented as part of pull-request #58

**Comment by wilsonpage** [comment]
Landed as an extension f8760dd51f6d7c6573771ecdd39850f1f045f9c0
